common/src/main/java/com/wynntils/core/keybinds/KeyBind.java

```diff
@@ -43,6 +43,10 @@ public KeyBind(String name, int keyCode, boolean firstPress, Runnable onPress, C
     public KeyBind(String name, int keyCode, InputConstants.Type type, boolean firstPress, Runnable onPress) {
         this.firstPress = firstPress;
         this.keyMapping = new KeyMapping(name, type, keyCode, CATEGORY);
+
+        // Unbound by default
+        keyMapping.setKey(InputConstants.UNKNOWN);
+        KeyMapping.resetMapping();
```

Comment:
Why? Don't we want to actually register the keybind? This feels smth like "@StartDisabled" for keybinds. I think the changes in `unregisterKeybind` should be enough

Reply:
This should unbind the keybind, and if the keybind is registered it should be bound by the options loading later.

Reply:
Actually, registering keybind should bind it too, oops